pyoverkiz/enums/server.py

```diff
@@ -0,0 +1,26 @@
+from enum import Enum, unique
+
+
+@unique
+class APIType(str, Enum):
+    CLOUD = "cloud"
+    LOCAL = "local"
+
+
+@unique
+class Server(str, Enum):
```

Comment:
@tetienne I already added this one from your PR. Is a great add and would be good to have this in soon, before we fully migrate to the "v2" style of this client.

Reply:
Perfect, I will see if I can backport more changes without breaking anything